tests/acceptance/01_vars/02_functions/findfiles.cf

```diff
@@ -76,27 +72,21 @@ bundle agent test
 bundle agent check
 {
   vars:
-      "expected[a]" string => "$(G.testdir)/a,$(G.testdir)/d,$(G.testdir)/g";
-    windows::
-      "expected[b]" string => "$(G.testdir)/a,$(G.testdir)/bc,$(G.testdir)/d,$(G.testdir)/g,$(G.testdir)/klm";
-    !windows::
-      "expected[b]" string => "$(G.testdir)/a,$(G.testdir)/bc,$(G.testdir)/d,$(G.testdir)/g,$(G.testdir)/klm,$(G.testdir)/tu";
+      "expected[a]" string => "$(G.testdir)$(const.dirsep)a,$(G.testdir)$(const.dirsep)d,$(G.testdir)$(const.dirsep)g";
+      "expected[b]" string => "$(G.testdir)$(const.dirsep)a,$(G.testdir)$(const.dirsep)bc,$(G.testdir)$(const.dirsep)d,$(G.testdir)$(const.dirsep)g,$(G.testdir)$(const.dirsep)klm,$(G.testdir)$(const.dirsep)tu";
 
-    any::
-      "expected[c]" string => "$(G.testdir)/d/e,$(G.testdir)/g/h";
-      "expected[d]" string => "$(G.testdir)/a,$(G.testdir)/bc";
+      "expected[c]" string => "$(G.testdir)$(const.dirsep)d$(const.dirsep)e,$(G.testdir)$(const.dirsep)g$(const.dirsep)h";
+      "expected[d]" string => "$(G.testdir)$(const.dirsep)a,$(G.testdir)$(const.dirsep)bc";
       "expected[e]" string => "";
 
-    !windows::
-      "expected[f]" string => "$(G.testdir)/tu/*";
-      "expected[g]" string => "$(G.testdir)/a,$(G.testdir)/bc,$(G.testdir)/d,$(G.testdir)/g,$(G.testdir)/klm,$(G.testdir)/tu,$(G.testdir)/d/e,$(G.testdir)/g/h,$(G.testdir)/klm/nop,$(G.testdir)/tu/*,$(G.testdir)/d/e/f,$(G.testdir)/g/h/i,$(G.testdir)/klm/nop/qrs,$(G.testdir)/g/h/i/j";
-      "expected[h]" string => "$(G.testdir)/g/h/i/j";
+      "expected[f]" string => "$(G.testdir)$(const.dirsep)tu$(const.dirsep)*";
+      "expected[g]" string => "$(G.testdir)$(const.dirsep)a,$(G.testdir)$(const.dirsep)bc,$(G.testdir)$(const.dirsep)d,$(G.testdir)$(const.dirsep)g,$(G.testdir)$(const.dirsep)klm,$(G.testdir)$(const.dirsep)tu,$(G.testdir)$(const.dirsep)d$(const.dirsep)e,$(G.testdir)$(const.dirsep)g$(const.dirsep)h,$(G.testdir)$(const.dirsep)klm$(const.dirsep)nop,$(G.testdir)$(const.dirsep)tu$(const.dirsep)*,$(G.testdir)$(const.dirsep)d$(const.dirsep)e$(const.dirsep)f,$(G.testdir)$(const.dirsep)g$(const.dirsep)h$(const.dirsep)i,$(G.testdir)$(const.dirsep)klm$(const.dirsep)nop$(const.dirsep)qrs,$(G.testdir)$(const.dirsep)g$(const.dirsep)h$(const.dirsep)i$(const.dirsep)j";
```

Comment:
I gotta wonder if `$(const.dirsep)` isn't the best idea... this policy is rather unreadable.

Maybe `/` being the standard and on windows we replace with `$(const.dirsep)` might be a better idea. Or even `\` replaced with `/` on unix so go either way. :p